doc/MAGIC.md

```diff
@@ -692,7 +721,16 @@ The following is a list of possible `values`:
 | `STRENGTH` | 
 | `SPEED` | 
 | `EFFECTIVE_HEALTH_MOD` | If this is anything other than zero (which it defaults to) you will use it instead of your actual health mod.
+| `EXTRA_ACID` | 
+| `EXTRA_BASH` | Increases amount of pain when receiving bash damage.
```

Comment:
Are you sure it increase the pain? Not damage?

Reply:
You're right, it's extra damage received, not pain.
Tested with a new character, naked and wielding an enchanted item, enchantment with `EXTRA_BASH` value `"add": 9` and casting a spell dealing 1 bash dmg =  total 10 bash dmg

![2](https://user-images.githubusercontent.com/95231551/190919819-be1937c7-89f9-4008-93e5-f2c746a001b1.png)

![1](https://user-images.githubusercontent.com/95231551/190919817-5d99784b-a4d2-4a24-9815-05b64681376e.png)


Then changed `EXTRA_BASH` to `"multiply": 3` and the spell damage to 4 bash dmg = 4 + 300% = 16 bash dmg


![4](https://user-images.githubusercontent.com/95231551/190920273-c9b895b3-8f29-4424-9427-614c6f1501a9.png)

![3](https://user-images.githubusercontent.com/95231551/190920268-dc9c4351-3b1e-4b0b-af5e-45828c7ff7dd.png)